### ISSUE TITLE
Not working in 1.19

### ISSUE THREAD

**Issue by Artzert2215** [Rapha149/MessageHider#2]
**Describe the bug**
The plugin misses the class `net.minecraft.network.protocol.game.PacketPlayOutChat`?

**logs**
```
[16:10:55] [Netty Epoll Server IO #0/WARN]: java.lang.ClassNotFoundException: net.minecraft.network.protocol.game.PacketPlayOutChat
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:153)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:105)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at java.base/java.lang.Class.forName0(Native Method)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at java.base/java.lang.Class.forName(Class.java:375)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at MessageHider_1.8.3.jar//de.rapha149.messagehider.util.ReflectionUtil.getClass(ReflectionUtil.java:60)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at MessageHider_1.8.3.jar//de.rapha149.messagehider.Events$1.write(Events.java:66)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:764)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:790)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:758)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:1020)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:311)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at ProtocolLib_DEV-BUILD.jar//com.comphenix.protocol.injector.netty.channel.NettyChannelProxy.writeAndFlush(NettyChannelProxy.java:227)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at ProtocolLib_DEV-BUILD.jar//com.comphenix.protocol.injector.netty.channel.NettyChannelProxy.writeAndFlush(NettyChannelProxy.java:233)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at net.minecraft.network.NetworkManager.doSendPacket(NetworkManager.java:463)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at net.minecraft.network.NetworkManager.lambda$writePacket$11(NetworkManager.java:437)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:391)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:995)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[16:10:55] [Netty Epoll Server IO #0/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:833)
```

**Additional information (please complete the following information):**
 - Server version 1.19
 - Java version 17
 - Plugin version 1.8.3

**Comment by Artzert2215** [comment]
class changed: https://www.spigotmc.org/threads/packetplayoutchat-missing.560356/

**Comment by Rapha149** [comment]
I'm aware of the issue, but thanks for reporting anyway!

As you already mentioned, the class has changed and it was also split into two classes. I have yet to test how these classes are filled unter certain circumstances, but I'm pretty sure I will have to redesign the whole plugin in order to update to 1.19.
That said, it might take a while, until I'm done.

**Comment by Rapha149** [comment]
Support for 1.19 is now added, but there are a few things that could be problematic which I hope get fixed in 1.19.1. I have described these in the Github release and the update on Spigot.
